pkg/translator/loki/logs_to_loki.go

```diff
@@ -297,8 +296,13 @@ func addLogLevelAttributeAndHint(log plog.LogRecord) {
 }
 
 func addHint(log plog.LogRecord) {
-	if value, found := log.Attributes().Get(hintAttributes); found && !strings.Contains(value.AsString(), levelAttributeName) {
-		log.Attributes().PutStr(hintAttributes, fmt.Sprintf("%s,%s", value.AsString(), levelAttributeName))
+	if value, found := log.Attributes().Get(hintAttributes); found {
+		switch value.Type() {
+		case pcommon.ValueTypeSlice:
+			value.Slice().AppendEmpty().SetStr(levelAttributeName)
+		case pcommon.ValueTypeStr:
+			log.Attributes().PutStr(hintAttributes, value.AsString()+","+levelAttributeName)
```

Comment:
I'm surely missing something, but why didn't it work with fmt.Sprintf?

Reply:
it worked, moved `fmt.Sprintf` back to decrease the diff